src/containers/Tabs/Account.tsx

```diff
@@ -53,12 +54,20 @@ const AccountTabContainer: React.FC<AccountTabContainerProps> = ({ isOpen }) =>
                 toggleModal("add")();
                 addTransaction(value, PENDING_ADDITION);
             })
-            .on("receipt", () => {
-                openToast(
-                    value,
-                    SUCCESS,
-                    `New whitelisted account processed: ${value}`
-                );
+            .on("receipt", (receipt: any) => {
+                if(idx(receipt, _ => _.events.AccountAdded.returnValues.accountAdded)) {
```

Comment:
Would also check whether there is an AccountAdded event at all and treat that as an unexpected error condition